spec/fixtures/static_pages/mp.php?mpn=Christine_Milne&mpc=Tasmania&house=senate&display=everyvote.html

```diff
@@ -106,14 +106,14 @@ <h2 class="social-share-heading small">
 
 <h2 id="divisions">All Votes</h2>
 <p>All votes Senator Christine Milne could have attended.</p>
-<ol class="display-house-all divisions-list list-unstyled">
+<ol class="display-house-senate divisions-list list-unstyled">
```

Comment:
Yeap. I am aware of that. What I was trying to say is that I tried to track down and I didn't manage to figure out why that changed. Sorry if I wasn't clear enough.